evennia/accounts/tests.py

```diff
@@ -66,7 +68,68 @@ def setUp(self):
         self.s1 = MagicMock()
         self.s1.puppet = None
         self.s1.sessid = 0
-        self.s1.data_outj
```

Comment:
This got merged in somewhere along the lines; appears to be an error? Removed it. Tests still pass...